sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1186,11 +1186,19 @@ def split(self, desired_bundle_size, start_position=None, stop_position=None):
           parent=parent,
           read_session=requested_session,
           max_stream_count=stream_count)
+      if self.use_native_datetime:
+        display_schema = "Arrow Schema:" + str(read_session.arrow_schema)
+      else:
+        display_schema = "Avro Schema:" + str(read_session.avro_schema)
       _LOGGER.info(
```

Comment:
this is to make this log readable. Before it was printing raw proto message truncated on streams URLs. Now it only prints the number of streams returned and the key fields.